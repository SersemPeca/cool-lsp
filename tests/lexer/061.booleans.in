true
tRue
trUe
truE
tRUe
trUE
tRuE
tRUE
