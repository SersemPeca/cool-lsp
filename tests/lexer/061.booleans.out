#1 BOOL_CONST true
#2 BOOL_CONST true
#3 BOOL_CONST true
#4 BOOL_CONST true
#5 BOOL_CONST true
#6 BOOL_CONST true
#7 BOOL_CONST true
#8 BOOL_CONST true
